Calibration/TkAlCaRecoProducers/python/ALCARECOPromptCalibProdSiPixelLorentzAngleMCS_cff.py

```diff
@@ -26,6 +26,7 @@
 from RecoTracker.Configuration.RecoTrackerP5_cff import *
 from RecoTracker.TrackProducer.TrackRefitter_cfi import *
 from TrackingTools.TransientTrack.TransientTrackBuilder_cfi import *
+from RecoLocalTracker.SiPixelRecHits.SiPixelTemplateStoreESProducer_cfi import *
```

Comment:
I should have had commented already in #42597...

But as of now these `include`'s in the modified Calibration/TkAlCaRecoProducers/python configs are not needed any more. Should they get removed, or do you plan to let them there as a placeholder for future reinsertion of Tasks in them?

Reply:
I don't have any (short term) plans on reinserting Tasks.

Reply:
> I don't have any (short term) plans on reinserting Tasks.

Thank you @Dr15Jones 
Should we then clean up these backports from those unnecessary includes, together with the master (with an additional PR)?

Reply:
yes - please.